src/services/api.tsx

```diff
@@ -0,0 +1,57 @@
+import { URL_SERVER } from "../constants";
+import { Patient } from "types";
+
+const mkFhir = require("fhir.js");
+const client = mkFhir({
+  baseUrl: URL_SERVER
+});
+
+function getAge(birthDate: Date) {
+  const today = new Date();
+  const monthDifference = today.getMonth() - birthDate.getMonth();
+
+  const age = today.getFullYear() - birthDate.getFullYear();
+  if (
+    monthDifference < 0 ||
+    (monthDifference === 0 && today.getDate() < birthDate.getDate())
+  )
+    return age - 1;
+  return age;
+}
+
+const callApi = (resourceType: string, queryParameters: object) => {
```

Comment:
I think it would be clearer to remove this function and use client.search directly in the `getPatients` function. What do you think ?